### ISSUE TITLE
SRAF requiring SRTS Components

### ISSUE THREAD

**Issue by NicJames2378** [Neceros/SRTS-Expanded#28]
I don't think this is really a bug, so much as an oversight. While being capable of researching "The SRAF" before "The Skip", the SRAF requires specialized building materials only unlocked when you research the Skip. 

I propose either making the components their own research item to go before the SRAF (and subsequently making the SRAF require it, basically shifting the entire lineup for larger ships), or simply making the SRAF require the Skip to be researched first (change dependency from "Research_SRTSSuperpod" to "Research_SRTSSkip"). Not really a bug but it caused me to spend a good hour trying to debug why the components weren't in fabrication benches!

**Comment by NicJames2378** [comment]
I've made two pull requests with my mentioned fixes (since it's not a bug, I figured I'd save you time on it since you make so many of the mods I spend all of my time in :D). The first is #29 which will change one line for the requirements. The second is #30 which will change a bit more, such as adding the components research item, restructuring the research items for people who use the vanilla research manager instead of Fluffy's, and corrects some grammatical errors in the flavor text. Nothing that would break anything, just some QoL things.
